Vulnerability Detector: NVD API 2.0 (#6247)
wazuh/wazuh-documentation: source/user-manual/capabilities/vulnerability-detection/offline-update.rst

National Vulnerability Database
-------------------------------

To perform an offline update of the National Vulnerability Database, you must get the feed from `complete_nvd_feed.json.gz <https://feed.wazuh.com/vulnerability-detector/NVD/generated-feeds/complete_nvd_feed.json.gz>`__ (`sha256 <https://feed.wazuh.com/vulnerability-detector/NVD/generated-feeds/complete_nvd_feed.meta>`__) and place it in a user-defined repository.

.. code-block:: xml

   <provider name="nvd">
      <enabled>yes</enabled>
      <url>http://local_repo/complete_nvd_feed.json.gz</url>
      <update_interval>1h</update_interval>
   </provider>

> To update locally, you have to set the path of the downloaded files with a POSIX regular expression as such:

Reviewer: ```suggestion
To update locally, set the path of the downloaded files using a POSIX regular expression, for example:
```

Author: Should I also change the MSU section? I copy paste the description from there:

https://github.com/wazuh/wazuh-documentation/blob/74d70888e9305f077478b1037a4929f3905a43ad/source/user-manual/capabilities/vulnerability-detection/offline-update.rst?plain=1#L399

Reviewer: It's not necessary to merge this PR, but it's a good opportunity to improve the text if you want to.
